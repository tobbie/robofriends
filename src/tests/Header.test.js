import { shallow } from "enzyme";
import React from "react";
import Header from "../components/Header";

describe("searchbox tests", () => {
  it("expect to render header component", () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
