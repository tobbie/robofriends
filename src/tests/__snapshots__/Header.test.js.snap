// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchbox tests expect to render header component 1`] = `ShallowWrapper {}`;
